frontend/src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -120,29 +116,7 @@ export const ProjectAccessTable: VFC = () => {
                 maxWidth: 85,
                 disableSortBy: true,
             },
-            {
-                id: 'name',
-                Header: 'Name',
-                accessor: (row: IProjectAccess) => row.entity.name || '',
-                Cell: ({ value, row: { original: row } }: any) => (
-                    <ConditionallyRender
-                        condition={row.type === ENTITY_TYPE.GROUP}
-                        show={
-                            <LinkCell
-                                onClick={() => {
-                                    setSelectedRow(row);
-                                    setGroupOpen(true);
-                                }}
-                                title={value}
-                                subtitle={`${row.entity.users?.length} users`}
-                            />
-                        }
-                        elseShow={<HighlightCell value={value} />}
-                    />
-                ),
-                minWidth: 100,
-                searchable: true,
-            },
+
```

Comment:
Would this not break displaying of group?

Reply:
Currently looks like this
![image](https://user-images.githubusercontent.com/964450/201322692-91496480-29ec-49da-9304-25d849e99fd9.png)